app/decorators/openid_connect_authorize_decorator.rb

```diff
@@ -1,22 +1,15 @@
 class OpenidConnectAuthorizeDecorator
-  attr_reader :scopes, :service_provider
+  attr_reader :requested_attributes
 
-  delegate :metadata, to: :service_provider
-
-  def initialize(scopes:, service_provider:)
+  def initialize(scopes:)
     @scopes = scopes
-    @service_provider = service_provider
-  end
-
-  def name
-    metadata[:friendly_name] || metadata[:agency]
   end
 
   def requested_attributes
     OpenidConnectAttributeScoper.new(scopes).requested_attributes
```

Comment:
this line is the only thing this class does anymore, should we just remove the class entirely and write a `@requested_attributes` var directly in the controller?